server/models/Expense.js

```diff
@@ -498,6 +510,44 @@ function defineModel() {
     );
   };
 
+  Expense.getCollectiveExpensesTagsTimeSeries = async function (
+    collectiveId,
+    timeUnit,
+    { dateFrom = null, dateTo = null } = {},
+  ) {
+    const noTag = 'no tag';
+    return sequelize.query(
+      `
+      SELECT DATE_TRUNC($timeUnit, t."createdAt") AS "date",
+      TRIM(UNNEST(COALESCE(e."tags", '{"${noTag}"}'))) AS label,
+      COUNT(e."id") as "count",
+      ABS(SUM(t."amountInHostCurrency")) as "amount",
```

Comment:
We should convert this to collective currency.

Reply:
Now looking at collective `currency`